recipes/epysurv/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "epysurv" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # If getting the source from GitHub remove the line above
+  # uncomment the line below and modify as needed
+  url: https://github.com/jarnorfb/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 56383ce024e2ad76c1ac300065efe555cc57fcbc8d7ab00052b310195470e80c
+
+build:
+  # noarch: python
+  number: 0
+  merge_build_host: True  # [win]
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv"
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    # if your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('cxx') }}
+  host:
+    - python=3.7
```

Comment:
This package won't work on python 3.6 as well?

Reply:
The package makes use of dataclasses. But now that I think about it, it should be fine to add dataclasses as a dependency for python 3.6, which should make it work in 3.6 as well.